AK/bits/Math/libm_fallback.h

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2021, Leon Albrecht <leon2002.la@gmail.com>.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Concepts.h>
+#include <AK/Types.h>
+#include <math.h>
```

Comment:
We can’t include math.h in Kernel mode?

Reply:
Math.h should only be used in Userland
and the libm_fallback is for lagom on non-x86 platforms only, otherwise it would call back to itself.

Reply:
(also I am bad at naming....)